Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -170,24 +141,63 @@ extension FeatureFormExampleView {
     /// A Boolean value indicating whether general form workflow errors are presented.
     private var alertIsPresented: Binding<Bool> {
```

Comment:
This should just be a state property. Which an error is presented and which error is presented are two different properties of the view's state.

Reply:
@mhdostal, any preference between the two approaches? The fewer lines of code is nice...